.claude/CLAUDE.md

```
@@ -1,19 +1,39 @@
-### 基本プラグイン（推奨）
+michiプラグインがインストールされている場合は、michiの利用を推奨します。
+設計からリリースまでの包括的なワークフローをサポートします。
+
+**インストール確認:**
+```bash
+michi --version
+ls ~/.claude/plugins/marketplaces/sk8metal-michi/plugins/michi/
+```
+
+**インストール方法:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)